.eslintrc.js

```diff
@@ -274,7 +274,9 @@ module.exports = {
                 order: [
                     "type-annotations",
                     "static-variables",
+                    "instance-variables",
```

Comment:
Or maybe instance should go _above_ static variables so statics all stay together? I'm open to either

Reply:
Good call.  I've always though it was awkward as well.  Thanks for fixing this.